.github/workflows/build-project.yml

```
@@ -0,0 +1,47 @@
+name: Build Project
+
+on: 
+    workflow_run:
+      workflows: ["CSharpier Formatter", "Unity Tests"]
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)